### PR TITLE
Feature/Update dashboard base components styling

### PR DESCRIPTION
**Pull request by Tony9984**
### Tickets:

- HCK-#864

### List of changes:
Update styling of the following shared components:
 - [x] Buttons
 - [x] Checkboxes
 - [x] Text inputs
 - [x] Text area
 - [x] Select/multi-select

### Type of change:

Please delete options that aren't relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### PR Checklist:

- [x] Merged `develop` branch (before testing)
- [x] Linted my code locally
- [ ] Listed change(s) in the Changelog
- [x] Tested all links in project relevant browsers
- [x] Tested all links on different screen sizes
- [x] Referenced all useful info (issues, tasks, etc)

### Screenshots:

Primary & Secondary Buttons [!isOutlined]
image

Primary & Secondary Buttons [isOutlined]
image

Checked & Unchecked checkbox
image

Focused select
image

Focused text input
image

### CODE REVIEW

File: src/shared/Form/FormikElements/Checkbox.tsx [hunk 1]
```diff
@@ -22,6 +22,7 @@ const CheckboxContainer = styled.div`
     border-radius: 4px;
     cursor: pointer;
     outline: none;
+    box-shadow: 0px 0px 16px 0px rgba(16, 20, 55, 0.1);
```
Comment (logan-r) [inline src/shared/Form/FormikElements/Checkbox.tsx:25]: nit: color should be from theme constant

File: src/shared/Styles/theme.ts [hunk 2]
```diff
@@ -10,6 +10,9 @@ const theme = {
     teal: '#48DEE2',
     tealLight: '#88FCFF',
 
+    purple: '#5C63AB',
+    purpleLight: 'rgba(92, 99, 171, 0.1)',
```
Comment (logan-r) [inline src/shared/Styles/theme.ts:14]: nit: change to #5C63AB19 for consistency

**Review by logan-r — CHANGES_REQUESTED** [review]
Looks great 🔥

Just the small changes listed and also fill out screenshots section

**Review by logan-r — CHANGES_REQUESTED** [review]
Good good

Could you also get rid of the border along the text inputs and increase border radius of buttons so they match the figma?

**Comment by logan-r** [comment]
We're going to make the background an offwhite to increase the contrast and we'll probably end up making the box-shadow less pronounced then, but that can wait until after the backgrounds implemented so we can see how it looks

**Comment by Tony9984** [comment]
> Good good
> 
> 
> 
> Could you also get rid of the border along the text inputs and increase border radius of buttons so they match the figma?

Yeah sure!

**Review by logan-r — APPROVED** [review]
Good to merge!

**Comment by logan-r** [comment]
Also when squashing and merging, have the message be something like "feat: update dashboard base components styling" as we use [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)

**Pull request merged**